assets/aws/files/tests/fixtures/common.bash

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+export TELEPORT_TEST_MODE=true
+export TELEPORT_TESTVAR_LOCAL_IP=10.1.2.3
+export TELEPORT_TESTVAR_LOCAL_HOSTNAME=ip-10-1-2-3.ec2.internal
+export TELEPORT_TESTVAR_PUBLIC_IP=1.2.3.4
+
+TEST_SUITE="$(basename ${BATS_TEST_FILENAME%%.bats})"
+
+setup() {
+    load fixtures/test-setup
+    load fixtures/${TEST_SUITE?}
```

Comment:
can test-specific fixtures be created inside the test itself?
it's easier to scroll up in a file to see input data, rather than looking in a separate file

Reply:
I actually did this in the first iteration but it seemed less tidy. I see the benefit of doing it though, I'll give it a go.